Dockerfile

```diff
@@ -2,10 +2,39 @@ FROM python:3.6-alpine
 
 LABEL version="0.1.0"
 
-# https://github.com/aws/aws-cli/blob/master/CHANGELOG.rst
-ENV AWSCLI_VERSION='1.16.237'
+ENV GLIBC_VER=2.31-r0
+ENV AWS_CLI_VER=2.3.6
 
-RUN pip install --quiet --no-cache-dir awscli==${AWSCLI_VERSION}
+# We need AWS CLI v2 but it isn't compatible out-of-the-box with alpine so we
+# have to do this magic.
+# For more info: https://stackoverflow.com/questions/61463584/docker-image-with-aws-cli-v2-and-dind-based-on-alpine3-11
+RUN apk --no-cache add \
```

Comment:
Another option, apparently, is to switch to using `ubuntu` as the base image. I only went with this solution because it doesn't change the base image but there's no particular reason I wanted to keep the base image the same -- happy to switch to using `ubuntu` if there's a reason.

Reply:
don't feel strongly here. If I were starting fresh, would probably use ubuntu since that giant curl mess looks gross but I'm not going to advocate deleting it :P